04.ls/ls_command.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+COLUMN_WIDTH = 3
+
+def main
+  files_array = fetch_files_array
+  puts format_to_show(files_array)
+end
+
+def fetch_files_array
+  files_array = []
+  dir_path = configure_dir
+  Dir.foreach(dir_path) do |file|
+    next if file.start_with?('.')
+
+    files_array	<< file
+  end
+  files_array
+end
+
+def configure_dir
+  ARGV[0] ||= '.'
```

Comment:
ARGVを勝手に変更してしまうのはあまりお行儀が良くないですね。（ARGVはグローバルな値なので）
そもそもここは

```suggestion
  ARGV[0] || '.'
```

で良かったりしませんか？

あと、configureは設定みたいな意味で使われるので、この処理とちょっとマッチしない気がしました。